components/password.jsx

```diff
@@ -0,0 +1,70 @@
+import React from 'react';
+import { PropTypes } from 'prop-types';
+import classNames from 'classnames';
+
+export default function Password ({
+	unknownUser = false,
+	hasError = false,
+	pattern = null,
+	isDisabled = false,
+	fieldId = 'passwordField',
+	inputId = 'password',
+	inputName = 'password',
+}) {
+
+	const PasswordFieldClassNames = classNames([
+		'o-forms o-forms--wide',
+		'ncf__field',
+		'js-field',
+		{
+			'o-forms--error': hasError,
+			'js-unknown-user-field': unknownUser
+		}
+	]);
+
+	return (
+		<div
+			id={fieldId}
+			className={PasswordFieldClassNames}
+			data-ui-item="form-field"
+			data-ui-item-name="password"
+			data-validate="required,password"
+		>
+
+			<label htmlFor="password" className="o-forms__label">Password</label>
```

Comment:
Need to make sure that `htmlFor` points to the correct input, so:
```suggestion
			<label htmlFor={inputId} className="o-forms__label">Password</label>
```

Reply:
Good spot.